recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - double-conversion
+        - flex
+        - fmt
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libclang-cpp {{ llvm_version }}
+        - libgdal
+        - libpng
+        - librdkafka
+        - libzlib
+        - tbb
+        - tbb-devel
+        - thrift-cpp {{ thrift_version }}
+        - xerces-c
+      run:
+        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
+        - boost-cpp
+        - bzip2
+        # omnscidb Load-time UDF support calls clang++
+        - gxx_{{ target_platform }}
+        - {{ pin_compatible('libclang-cpp', max_pin='x.x') }}
+        - ncurses
+        - openjdk 8.*
+        - xz
+        - zlib
+        - {{ pin_subpackage('heavydb-common',  max_pin='x.x.x') }}
+      run_constrained:
+        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
+        - cudatoolkit >=11.0  # [cuda_compiler_version not in (undefined, "None")]
+
+    test:
+      commands:
+        # Test installation
+        - test -f ${PREFIX}/lib/libDBEngine.so
+        # Verify the activation scripts are in-place.
+        {% for state in ["activate", "deactivate"] %}
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
+        {% endfor %}
+        - |
+          source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
+          test -n "${HEAVYAI_ROOT_PATH+x}"
+          ldd ${PREFIX}/lib/libDBEngine.so
+
+    about:
+      home: https://heavy.ai
+      license: Apache-2.0
+      license_family: APACHE
+      license_file: LICENSE.md
+      summary: The HeavyDB database
+
+      description: |
+        HeavyDB is an in-memory, column store, SQL relational database
+        that was designed from the ground up to run on GPUs.
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
+
+  - name: pyheavydbe
+    version: {{ version }}
+    script: build-heavydb.sh
+    build:
+      string: {{ py_build_string_ext }}
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+      skip: true  # temporarily skipping pyheavydbe build
+      missing_dso_whitelist:
+        - '*/libcuda.*'  # [cuda_compiler_version not in (undefined, "None")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      ignore_run_exports:
+        - arrow-cpp {{ arrow_version }}
+        - blosc
+        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
+        - double-conversion
+        - fmt
+        - glog
+        - libarchive
+        - libgdal
+        - libllvm{{ llvm_version }}
```

Comment:
intention no blank ?

Reply:
As I said in this https://github.com/conda-forge/staged-recipes/pull/18691#issuecomment-1112241818, we did not touch much at the recipe because it's just a rebranding of an existing recipe. Everything is quite fragile so we tried to not do too much. E.g. windows build is not working here hence the skip. In a later process we will update things and reactivate things.

Reply:
I did a package search on ubuntu, examples: libllvm6.0, libllvm7, ... so yes, that is intended to concat the version